tests/features/collections/newspaper.feature

```diff
@@ -45,10 +45,3 @@ Feature: Newspapers collection
     And I go to page number 2
     And I wait for the page to load
     Then I should be on `/en/collections/topic/18-newspapers?page=2&view=grid&api=metadata`
-
-  Scenario: Newspapers collection API toggle is removed by removing search pill
```

Comment:
This should be replaced by a test that checks the removal by using the 'Search for "query" in our entire Collection' option.

Reply:
I can't find this scenario

Reply:
Something like this I mean:
```
 Scenario: Newspapers collection API toggle is removed by searching in the entire collection
    Given I am on `/en/collections/topic/18-newspapers?api=fulltext&view=grid`	
    When I click the `show search button`	
    And I enter "paris" into the `search box`
    And I click the `search entire collection button`	
    Then I see the `search page`	
    And I should be on `/en/search?page=1&view=grid&query=paris`
```
Might need a bit of tweaking for the order of the search params on the final page, but the important thing is that it doesn't contain 'api=fulltext'.

Reply:
ok